src/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-case-declarations */
+/* eslint-disable consistent-return */
```

Comment:
we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.
Kindly do the same in all your codebase, I can't comment directly on your code with the way you write/push your code.

**_In the next milestone, your PR might not be approved because of inline comment._**  👀  **on you.**